src/Shellbar/__stories__/Shellbar.stories.js

```diff
@@ -198,7 +198,12 @@ export const coPilot = () => (
         profileMenu={profileMenu}
         searchInput={{
             label: 'Search',
-            placeholder: 'Search'
+            placeholder: 'Search',
+            popoverProps: {
+                //to always show search-results popup towards bottom
+                disableEdgeDetection: true,
+                placement: 'bottom'
+            }
```

Comment:
Should we add these as defaults (in shellbar only - not searchinput in general) for consumers that they can then override if they want to?

Reply:
Makes sense